turbopack/crates/turbopack-ecmascript/src/utils.rs

```diff
@@ -27,16 +28,24 @@ pub fn unparen(expr: &Expr) -> &Expr {
 pub fn js_value_to_pattern(value: &JsValue) -> Pattern {
     let mut result = match value {
         JsValue::Constant(v) => Pattern::Constant(match v {
-            ConstantValue::Str(str) => str.as_str().into(),
-            ConstantValue::True => "true".into(),
-            ConstantValue::False => "false".into(),
-            ConstantValue::Null => "null".into(),
+            ConstantValue::Str(str) => {
+                // Normalize windows file paths when constructing the pattern.
```

Comment:
This will rewrite any string though, not just the paths?

Reply:
correct.

my understanding of the Pattern classes is it is used to match against 'require-specifiers' which are 'path-like'  so this seemed plausible, but as discussed on slack i am uncertain.

i could easily pass a `normalize_windows_paths` parameter... would that be better, or do you think i should move this logic somewhere else?

Reply:
On second thought, `js_value_to_pattern` is only called on path-like requests, so this is the right place.